demo/src/index.js

```diff
@@ -58,6 +59,14 @@ const Playground = () => (
             </Box>
           </Space>
         </Space>
+        <Space mt={6}>
+          <Box>
+            <Typography variant='titleLarge'>Tooltips</Typography>
+            <TooltipExamples>
```

Comment:
The `*Examples` components shouldn't receive props because they don't have customizable behaviour. They are like a `screen`